src/client/components/Bruincast/index.js

```diff
@@ -74,6 +87,37 @@ export const Bruincast = ({
   };
   useEffect(retrieveCasts, []);
 
+  const [analytics, setAnalytics] = useState(null);
+  const retrieveAnalytics = () => {
+    if (userIsInstructor()) {
```

Comment:
@maxpluez We want to also allow admins to view analytics

Reply:
userIsInstructor() contains admin if you look at its implementation. I can change the name of the function to make it clearer.

Reply:
@maxpluez You mean this? https://github.com/ucla/lti-media-resources/blob/50dfea74561b867bdea21d9bbaa21d583d41bcde/src/client/components/App/index.js#L66

I see it just checking teaching or instructor.

Maybe have a separate function called userIsInstructorOrAdmin()?